tests/integration/cattletest/core/test_resource_based_scheduling.py

```diff
@@ -95,6 +95,29 @@ def test_resource_based_scheduler(new_context, super_client):
                              'type': 'computePool'}])
 
 
+def test_delete_host(new_context, super_client):
+    host1 = new_context.host
+    host2 = register_simulated_host(new_context)
+    host3 = register_simulated_host(new_context)
+
+    mock_sched(new_context, super_client)
+    image = new_context.image_uuid
+    client = new_context.client
+
+    client.wait_success(host1.deactivate())
+    client.wait_success(host2.deactivate())
+    client.wait_success(host3.deactivate())
+    client.wait_success(super_client.delete(host1))
```

Comment:
You should not need super_client, you just need to get the updated version of the host. Can do it like this:
host1 = client.wait_success(host1.deactivate())
host1 = client.wait_success(client.delete(host1))
because wait_success returns the reloaded version of host